composer.json

```diff
@@ -23,7 +23,7 @@
         "twig/twig": ">=1.12",
         "php": ">=5.5.0",
         "mtdowling/cron-expression": "1.0.*",
-        "doctrine/collections": "~1.2",
+        "doctrine/orm": "@stable",
```

Comment:
I guess you maybe want to allow, say, `~2.3` or whatever version is needed? There's no guarantee Doctrine 3 wouldn't bring breaking changes, and we are currently supporting an EOL'd version of Symfony.